vc-verifier/js/src/constant/CredentialVerifierConstants.js

```diff
@@ -0,0 +1,6 @@
+export const CredentialVerifierConstants = {
+    SIGNATURE_SUITE_TERM: 'RsaSignature2018',
```

Comment:
can we rename this to something more meaningful?

Reply:
I have taken the constant names from the java library. Let's decide on the names and change it at both places to make it consistent across